site/_apps/nuxt-hn.txt

```diff
@@ -0,0 +1,29 @@
+---
+weight: 15
+title:  'HNPWA with Nuxt.js'
+github-title: 'nuxt/hackernews'
+libraries: 
+    - name: 'Vue'
+    - name: 'Nuxt'
+    - name: 'PWA module'
+module-bundling: 'Webpack'
+service-worker: 'Workbox'
+perfomance-patterns: 'PRPL, Prefetch/Preload JS + DNS + Data, Critical path CSS, Server side caching'
+server-side-rendering: 'Yes'
+api: 'Hacker News Firebase API'
+hosting: 'Now + CloudFlare CDN'
+authors: 
+    - name: 'Pooya'
+    - name: 'Sebastien'
+    - name: 'Alexandre'
+lighthouse: '91/100'
+interactive-em: '2.8s'
+interactive-faster-3g: '2.4s'
+lighthouse-link: https://www.webpagetest.org/lighthouse.php?test=170620_PG_a2a9feaf4ace07a61b2c6c2a171b1c79&run=1
+wpt-em-link: https://www.webpagetest.org/result/170620_AM_f5e1ae2f305c1d81b44cfe0c9c930e35
```

Comment:
The http->https problem is a known issue unfortunately, every implementation is impacted :(

See here for details: https://github.com/GoogleChrome/lighthouse/issues/2363